packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -45,6 +52,11 @@ void updateXcodeGeneratedProperties(String projectPath, BuildMode mode, String t
     localsBuffer.writeln('LOCAL_ENGINE=${localEngineArtifacts.engineOutPath}');
   }
 
+  // If the project isn't using platform plugins, don't assume CocoaPods will be
```

Comment:
It's correct, but a double-negative is hard to parse. Maybe "Only include CocoaPods dependency if the project is using plugins."?